src/openrct2/CmdlineSprite.cpp

```diff
@@ -36,22 +34,10 @@
 
 using namespace OpenRCT2::Drawing;
 
-#pragma pack(push, 1)
```

Comment:
There is a warning being disabled on line 10 (I can't really comment there). Maybe it's not applicable anymore? (I'm not sure, I haven't double checked)